docs/data-sources/csbs_backup_policy_v1.md

```diff
@@ -20,53 +20,63 @@ data "flexibleengine_csbs_backup_policy_v1" "csbs_policy" {
 
 The following arguments are supported:
 
-* `id` - (Optional) Specifies the ID of backup policy.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CSBS backup policy resource.
```

Comment:
1 `ForceNew` should be removed in data source;
2 the description is not suitable as it is a data source.

Reply:
thanks for your advice, I have fixed the problem